src/controlaudiotaperpot.h

```diff
@@ -12,7 +12,7 @@ class ControlAudioTaperPot : public ControlPotmeter {
     // minDB is the Start value of the pure db scale it cranked to -Infinity by the linear part of the AudioTaperPot
     // maxDB is the Upper gain Value
     // neutralParameter is a knob position between 0 and 1 where the gain is 1 (0dB)
-    ControlAudioTaperPot(ConfigKey key, double minDB, double maxDB, double neutralParameter);
+    ControlAudioTaperPot(ConfigKey key, double minDB, double maxDB, double neutralParameter = 0.0);
```

Comment:
I would prefer to set the neutralParameter=zeroDbParameter explicite, a default of 0.0 for the scale start works for me.